docs-2.0/3.ngql-guide/6.functions-and-expressions/2.string.md

```diff
@@ -76,8 +76,9 @@ nebula> RETURN upper("Basketball_Player");
 ## length()
 
 length() 返回：
-  - 指定字符串的长度，单位：字节。
-  - 路径的长度，单位：跳。
+
+- 指定字符串的长度，单位：字节。
```

Comment:
This is inconsistent with the definition of cypher's length function.
https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-length

Reply:
> This is inconsistent with the definition of cypher's length function. https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-length

Checked with @nevermore3, length() returns the length of a path or the length of a string.